src/backend/joanie/settings.py

```diff
@@ -118,13 +120,12 @@ class Base(Configuration):
     LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
 
     PARLER_LANGUAGES = {
-        None: (tuple(dict(code=code) for code, _name in LANGUAGES)),
+        SITE_ID: (tuple(dict(code=code) for code, _name in LANGUAGES)),
```

Comment:
It's because we make use of the "sites" contrib to build absolute urls for resource links and had to define  a `SITE_ID`. When doing so, the `PARLER_LANGUAGES` setting expects configurations for each site id.